src/main/resources/l10n/JabRef_fr.properties

```diff
@@ -1175,7 +1175,7 @@ Show_icons_for_groups=Afficher_les_icônes_pour_les_groupes
 Show_last_names_only=Afficher_uniquement_les_noms_propres
 
 Show_names_unchanged=Ordre_des_noms_inchangé
-
+        
```

Comment:
Looks like one of my changes. I noticed some spurious space characters in the French localization, and so removed them.
Now, looking at the English localization, these spaces are also present between the same keys:
```
Show_names_unchanged=Show_names_unchanged
        
Show_optional_fields=Show_optional_fields
```